presto-docs/src/main/sphinx/release/release-0.288.rst

```diff
@@ -0,0 +1,80 @@
+* Add Prestissimo support for CTAS into bucketed (but not partitioned) tables. :pr:`22737`
+* Add :doc:`/presto_cpp/properties` documentation. :pr:`22885`
+* Add PR number to the release note entry examples in pull_request_template.md. :pr:`22665`
+* Add ``http-server.authentication.allow-forwarded-https`` configuration property to recognize X-Forwarded-Proto header. :pr:`22492`
+* Add ``node-scheduler.max-preferred-nodes`` configuration property to allow changing number of preferred nodes when soft affinity scheduling is enabled. :pr:`22562`
+* Add documentation for :func:`!noisy_approx_set_sfm_from_index_and_zeros`. :pr:`22799`
+* Add documentation for noisy aggregate functions at :doc:`/functions/noisy`, including :func:`!noisy_approx_distinct_sfm` and :func:`!noisy_approx_set_sfm`. :pr:`22715`
+* Add support for memoizing in resource group state info endpoint. This can be enabled by setting ``cluster-resource-group-state-info-expiration-duration`` to a non-zero duration. :pr:`22764`
+* Add support for non default keystore and truststore type in presto CLI and JDBC. :pr:`22556`
+* Add support for querying system.runtime.tasks table in native clusters. :pr:`21416`
+* Remove deprecated feature and configuration property ``deprecated.group-by-uses-equal``, which allowed group by to use equal to rather than distinct semantics. :pr:`22888`
+* Upgrade CI pipeline to build and publish native worker docker image. :pr:`22806`
+* Upgrade Alluxio to 313. :pr:`22958`
+* Upgrade io.jsonwebtoken artifacts to 0.11.5. :pr:`22762`
+* Upgrade fasterxml.jackson artifacts to 2.11. :pr:`22417`
+
+Hive Connector Changes
+______________________
+* Fix hash calculation for Timestamp column to be hive compatible when writing to a table bucketed by Timestamp. :pr:`22980`
+* Improve affinity scheduling granularity from a file to a section of a file by adding a ``hive.affinity-scheduling-file-section-size`` configuration property and ``affinity_scheduling_file_section_size`` session property. The default file size is 256MB. :pr:`22563`
+* Add AWS Security Mapping to allow flexible mapping of Presto Users to AWS Credentials or IAM Roles for different AWS Services. :pr:`21622`
+* Add config property ``hive.legacy-timestamp-bucketing`` and session property ``hive.legacy_timestamp_bucketing`` to use the original hash function for Timestamp column, which is not hive compatible. :pr:`22980`
+
+Iceberg Connector Changes
+_________________________
+* Improve the partition specs that must be checked to determine if the partition supports metadata deletion or predicate thoroughly pushdown. :pr:`22753`
+* Improve time travel ``TIMESTAMP (SYSTEM_TIME)`` syntax to include timestamp-with-time-zone data type. :pr:`22851`
+* Improve time travel ``VERSION (SYSTEM_VERSION)`` syntax to include snapshot id using `BIGINT` data type. :pr:`22851`
+* Add procedure `expire_snapshots` to remove old snapshots in Iceberg. :pr:`22609`
+* Add support for Iceberg REST catalog. :pr:`22417`
```

Comment:
I would add this as a highlight

Reply:
@tdcmeehan how do I highlight a section? I could not find any instructions on the guidelines doc.

Reply:
They need to be repeated in the Highlights section above: https://github.com/prestodb/presto/pull/23079/files#diff-6d761c5d2106a4758cbb0b5efad14b1236e59241f80918ff7770ba6a9c7b21e4R5

Reply:
@wanglinsong I discussed how to do this with @tdcmeehan and I can populate the Highlights section.